pango.yaml

```diff
@@ -9,7 +9,6 @@ package:
   scriptlets:
     trigger:
       script: FIXME
-        - FIXME
```

Comment:
What's this about 🤨

Reply:
This comes from the converter, if an APKBUILD contains triggers then the converter adds a fixme so folks can review what the triggers are. https://github.com/chainguard-dev/melange/blob/3aaf78bfec504d2c97eec5ac440b64be2f98f701/pkg/convert/apkbuild/apkbuild.go#L351-L355

We could just remove this as scriptlets aren't supported in melange really, though sometimes it's good to be reminded to review the ones in Alpine.  

Maybe for now a wolfictl lint check `FIXME` comments would be a good thing.

Reply:
And I think the malformed YAML might have come from [here](https://github.com/wolfi-dev/os/commit/673302acc67fb3a50f526582f00e95b42959eeff#diff-de999f1a035dbf82a3b53590792b0073f081c03df5ea7d269cc049e8b7c14831) (after the converter ran).

I'd love to find consensus on the scriptlets usage, I think they cause some amount of confusion, personally.